schema/me/conversation/index.js

```diff
@@ -144,16 +158,29 @@ export const ConversationFields = {
 
   artworks: {
     type: new GraphQLList(ArtworkType),
-    resolve: conversation => {
-      const ids = []
-
-      for (const item of conversation.items) {
+    description: "Only the artworks discussed in the conversation.",
+    resolve: ({ items }) => {
+      const results = []
+      items.forEach(item => {
         if (item.item_type === "Artwork") {
-          ids.push(item.item_id)
+          results.push(item.properties)
         }
-      }
+      })
+      return results
+    },
+  },
 
-      return gravity("artworks", { ids })
+  items: {
+    type: new GraphQLList(ConversationItemType),
+    description: "The artworks and/or partner shows discussed in the conversation.",
+    resolve: ({ items }) => {
+      const results = []
+      items.forEach(item => {
+        if (item.item_type === "Artwork" || item.item_type === "PartnerShow") {
+          results.push(item.properties)
+        }
+      })
+      return results
```

Comment:
Well there's always going to be an item, since we don't delete `ConversationItem` records in Impulse (if the record has been truly deleted upstream).

We have a very small number of older conversations on actually deleted artworks, and several hundred show inquiries on presumably now-deleted fair booths. We also should never have that issue again due to the snapshotting in place.

So really, checking the `item_type` is appropriate (and limiting it to the two supported ones), and actually now that I mention those special cases, we should check that `properties` maybe has data?